### PR TITLE
MGMT-8313: Update event on Add Host to Installed Cluster

### PR DESCRIPTION
**Pull request by rollandf**
# Assisted Pull Request

## Description
Update event text on Add Host to Installed Cluster to redirect user to console.redhat.com

## List all the issues related to this PR

- [ ] New Feature
- [x] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [x] Cloud
- [ ] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Assignees

/cc @filanov 
/cc @nmagnezi 
/cc @nirfarkas 
## Checklist

- [x] Title and description added to both, commit and PR.
- [x] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] Reviewers have been listed
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [x] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: internal/cluster/cluster.go [hunk 1]
```diff
@@ -631,7 +631,7 @@ func (m *Manager) AcceptRegistration(c *common.Cluster) (err error) {
 	allowedStatuses := []string{models.ClusterStatusInsufficient, models.ClusterStatusReady, models.ClusterStatusPendingForInput, models.ClusterStatusAddingHosts}
 	if !funk.ContainsString(allowedStatuses, clusterStatus) {
 		if clusterStatus == models.ClusterStatusInstalled {
-			err = errors.Errorf("Cannot add host to a cluster that is already installed, please use the day2 cluster option")
+			err = errors.Errorf("Cannot add hosts to an existing cluster using the original Discovery ISO. Try to add new hosts by using the Discovery ISO that can be found in console.redhat.com under your cluster “Add hosts” tab.")
```
Comment (nmagnezi) [inline internal/cluster/cluster.go:634]:
```suggestion
			err = errors.Errorf("Cannot add hosts to an existing cluster using the original Discovery ISO. Try adding new hosts by using the Discovery ISO found in console.redhat.com under your cluster “Add hosts” tab.")
```

Comment (nmagnezi) [reply internal/cluster/cluster.go:634]: also, we have a knowledge of the cluster id here: maybe we can display the tab URL for this cluster directly?

File: internal/cluster/cluster_test.go [hunk 2]
```diff
@@ -1171,7 +1171,7 @@ var _ = Describe("VerifyRegisterHost", func() {
 		id                 strfmt.UUID
 		clusterApi         *Manager
 		preInstalledError  string = "Host can register only in one of the following states: [insufficient ready pending-for-input adding-hosts]"
-		postInstalledError string = "Cannot add host to a cluster that is already installed, please use the day2 cluster option"
+		postInstalledError string = "Cannot add hosts to an existing cluster using the original Discovery ISO. Try to add new hosts by using the Discovery ISO that can be found in console.redhat.com under your cluster “Add hosts” tab."
```
Comment (nmagnezi) [inline internal/cluster/cluster_test.go:1174]:
```suggestion
		postInstalledError string = "Cannot add hosts to an existing cluster using the original Discovery ISO. Try adding new hosts by using the Discovery ISO found in console.redhat.com under your cluster “Add hosts” tab."
```

**Review by nmagnezi — CHANGES_REQUESTED** [review]

File: internal/cluster/cluster.go (hunk 1 above)
Comment (rollandf) [reply internal/cluster/cluster.go:634]: OCM is using a different ID.

File: internal/cluster/cluster_test.go (hunk 2 above)
Comment (rollandf) [reply internal/cluster/cluster_test.go:1174]: I am using the text approved by @nirfarkas in the Jira ticket

**Review by nmagnezi — APPROVED** [review]
/lgtm

**Comment by nirfarkas** [comment]
Does this mean we need to show this only for SaaS clusters?

**Comment by rollandf** [comment]
/hold cancel
Updated to have different events if AI is running in SAAS

**Review by avishayt — APPROVED** [review]

**Pull request merged**